### PR TITLE
Feature/side bar

### PR DESCRIPTION
**Pull request by IvaskevychYuriy**

### CODE REVIEW

File: SimpleDrive.App/SimpleDrive.App.csproj [hunk 1]
```diff
@@ -26,10 +26,20 @@
...
+    <None Remove="ClientApp\components\Menu.tsx" />
+    <None Remove="ClientApp\components\SideMenu.tsx" />
+  </ItemGroup>
+
   <ItemGroup>
     <ProjectReference Include="..\SimpleDrive.DAL\SimpleDrive.DAL.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <TypeScriptCompile Include="ClientApp\components\Menu.tsx" />
```
Comment (nazar554) [inline SimpleDrive.App/SimpleDrive.App.csproj]: WTF?

File: SimpleDrive.App/SimpleDrive.App.csproj [hunk 2]
```diff
@@ -26,10 +26,20 @@
     <None Remove="_dbContext\**" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Remove="ClientApp\components\Menu.tsx" />
```
Comment (nazar554) [inline SimpleDrive.App/SimpleDrive.App.csproj]: This too)

**Review by nazar554 — CHANGES_REQUESTED** [review]
Pls fix .csproj

File: SimpleDrive.App/SimpleDrive.App.csproj (hunk 1 above)
Comment (IvaskevychYuriy) [reply SimpleDrive.App/SimpleDrive.App.csproj]: Fixed in 894a28c

File: SimpleDrive.App/SimpleDrive.App.csproj (hunk 2 above)
Comment (IvaskevychYuriy) [reply SimpleDrive.App/SimpleDrive.App.csproj]: Fixed in 894a28c

**Review by nazar554 — APPROVED** [review]

**Pull request merged**